ydb/src/credentials.rs

```diff
@@ -557,12 +559,15 @@ impl StaticCredentials {
         Ok(raw_response.token)
     }
 
-    pub fn new(username: String, password: String, endpoint: Uri, database: String) -> Self {
+    pub fn new(username: String,
```

Comment:
it will be better create new method: `new_with_ca` instead of modify method `new`:
- no break api
- allow use cert param only if need. It isn't need if connection doesn't need TLS or TLS has certificate from os-trusted ca.
-